app/src/main/res/values/strings.xml

```diff
@@ -355,7 +355,7 @@
     <string name="github_url" translatable="false">https://github.com/TeamNewPipe/NewPipe</string>
     <string name="view_on_github">View on GitHub</string>
     <string name="donation_title">Donate</string>
-    <string name="donation_encouragement">NewPipe is developed by volunteers spending time bringing you the best experience. Give back to help developers make NewPipe even better while enjoying a cup of coffee.</string>
+    <string name="donation_encouragement">NewPipe is developed by volunteers spending time bringing you the best experience. Give back to help make NewPipe even better.</string>
```

Comment:
```suggestion
    <string name="donation_encouragement">NewPipe is developed by volunteers spending their free time bringing you the best user experience. Give back to help developers make NewPipe even better while they enjoy a cup of coffee.</string>
```

removing the coffee here is imo not a good idea since the idea is that the cup of coffee IS THE DONATION also it's like that in other languages so it would be odd to only remove the cup of coffee here.

Reply:
Because it is tax exempt? I think people understand who are users, and who are developers. I don't understand the psychology you are going for, could you elaborate?

Reply:
so the idea is that coffee is cheap and people buy themselves coffee every day so to help out the project they can contribute by giving the equivalent of a coffee as a donation. The user / developers thing is not that important here.

Reply:
That idea doesn't work on people that have no relation to coffee. I agree that people spend more on things they hold dear, so it is indeed true the cost of a priority diminishes. My idea with that is to allude to it, and then play it into the added value by also tying it to NewPipe. 
"Treat the NewPipe makers to something you like, by way of funding its development."
Could personify "NewPipe makers" more, but I think this is better. Going with @cool-student version for now.